utils/constant/constants.go

```diff
@@ -347,8 +347,12 @@ const (
 const ServersDataSource = "ionoscloud_servers"
 
 const (
-	CertificateResource = "ionoscloud_certificate"
-	TestCertName        = "certTest"
+	CertificateResource             = "ionoscloud_certificate"
+	AutoCertificateProviderResource = "ionoscloud_auto_certificate_provider"
+	AutoCertificateResource         = "ionoscloud_auto_certificate"
+	TestCertName                    = "certTest"
```

Comment:
these seem to be used only in a single test file, is there really any reason for having them in the constants pkg

this pkg is imported many times in the code base

Reply:
@avirtopeanu-ionos it is also used in `import_certificate_manager_certificate_test.go`